Makefile

```diff
@@ -40,6 +46,9 @@ worker:
 cli:
```

Comment:
I think it's better to use 'make' for development and building, and for for production mode use directly docker-compose. But there is no problem to add them to Makefile